csss-site/src/announcements/templates/announcements/announcements.html

```diff
@@ -5,11 +5,20 @@
 {% block content %}
 	<link rel="stylesheet" href="{% static 'announcements_static/csss_custom.css' %}">
 	<div class="columns">
-		<div class="column is-three-quarters ">
+		<div class="column">
```

Comment:
I don't have a problem with this and it's a nice touch but if you are gonna add this, i feel like you should then remove the "SFU's Computing Science Student Society" banner, no?

![Screenshot from 2023-11-19 22-05-15](https://github.com/CSSS/csss-site/assets/14242088/609b9bbf-cbdf-4d18-8cd8-975da7dd2551)

just seems a bit much to me otherwise, thoughts?

Reply:
I see what you mean, that seems like a good idea! I'll try to combine the two "banners" so that they take up less vertical area

Reply:
![new-look](https://github.com/CSSS/csss-site/assets/24978329/18d97813-2cdf-4849-95cc-35f7fb026d45)
Here's my take now! I think it's better for the time being, though I'm sure Micah will revamp everything when he gets to it

Reply:
wow 👀 yall got a real eye for FE